modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/UserAccountResourcePerformanceTest.java

```diff
@@ -231,48 +233,20 @@ public void testMultipleThreadsAddUsers() throws Exception {
 			for (Future<?> future : futures) {
 				future.get();
 			}
-
-			long endTime = System.currentTimeMillis() - startTime;
-
-			if (_log.isInfoEnabled()) {
-				_log.info(
-					StringBundler.concat(
-						_threadsCount, " threads each added ",
-						usersCountPerThread, " users in ", endTime, " ms"));
-
-				double tps =
-					(double)usersCountPerThread * _threadsCount / endTime *
-						1000;
-
-				_log.info("TPS: " + String.format("%.2f", tps));
-			}
 		}
 	}
 
 	@Test
 	public void testSingleThreadAddUsers() {
 		List<String> jsons = _createJSONs(_usersCount);
 
-		try (LogCapture logCapture = LoggerTestUtil.configureLog4JLogger(
-				"com.liferay.mail.messaging.internal.MailMessageListener",
-				LoggerTestUtil.OFF)) {
-
-			long startTime = System.currentTimeMillis();
-
-			_addUsers(jsons);
-
-			long endTime = System.currentTimeMillis() - startTime;
-
-			if (_log.isInfoEnabled()) {
-				_log.info(
+		try (PerformanceTimer performanceTimer =
+				new UserAccountResourcePerformanceTimer(
+					_usersAddMaxTime,
 					StringBundler.concat(
-						"Single thread added ", _usersCount, " users in ",
-						endTime, " ms"));
+						"Single thread added ", _usersCount, " users"))) {
```

Comment:
@slnn Please adjust this following the usage in `ObjectEntryPerformanceTest`, like "Add _usersCount users with single thread", "Add _usersCountPerThread users per thread with _threadsCount threads". Actually, using a verb is still strange, because `PerformanceTimer` always appends "Starting" to the message. But since the pattern passed review in `ObjectEntryPerformanceTest` let's at least follow it strictly.